net/node/node.go

```diff
@@ -455,15 +455,21 @@ func (node *node) RemoveFlightHeightLessThan(h uint32) {
 
 func (node *node) RemoveFlightHeight(height uint32) {
 	log.Debug("height is ", height)
-	for _, h := range node.flightHeights {
-		log.Debug("flight height ", h)
-	}
 	node.flightHeights = SliceRemove(node.flightHeights, height)
-	for _, h := range node.flightHeights {
-		log.Debug("after flight height ", h)
-	}
 }
 
 func (node *node) GetLastRXTime() time.Time {
 	return node.time
 }
+
+func (node *node) StartConnTimer() {
+	t := time.NewTimer(time.Second * 10)
```

Comment:
this code will not work properly, since t was allocated each time node.Connect was called.  
case:  two Connect c1, c2 called at time 0s, 9s. then two timer will be created: t1, t2.   at time 10s, t1 is timeout,  which will remove c1, c2 at the same time .